pkg/controller/command/didclient/command.go

```diff
@@ -117,3 +123,41 @@ func (c *Command) CreateDID(rw io.Writer, req io.Reader) command.Error {
 
 	return nil
 }
+
+func (c *Command) SaveDIDToSDS(rw io.Writer, req io.Reader) command.Error {
+	didDataToStore := sdscomm.DIDData{}
+
+	err := json.NewDecoder(req).Decode(&didDataToStore)
+	if err != nil {
+		logutil.LogInfo(logger, commandName, saveDIDToSDSCommandMethod, "Failed to decode DID data: "+err.Error())
+
+		return command.NewValidationError(InvalidRequestErrorCode, fmt.Errorf("failed to decode DID data: %w", err))
+	}
+
+	saveDIDErr := c.saveDIDToSDS(&didDataToStore)
+	if saveDIDErr != nil {
+		return saveDIDErr
+	}
+
+	return nil
+}
+
+func (c *Command) saveDIDToSDS(didDataToStore *sdscomm.DIDData) command.Error {
+	sdsComm := sdscomm.New(c.sdsServerURL, c.agentUsername)
+
+	err := sdsComm.CreateDIDVault()
+	if err != nil {
+		logutil.LogInfo(logger, commandName, saveDIDToSDSCommandMethod, "Failed to create DID vault: "+err.Error())
+
+		return command.NewValidationError(InvalidRequestErrorCode, fmt.Errorf("failed to create DID vault: %w", err))
+	}
```

Comment:
Since a new `Command` object is created every time the "create and save DID" button is created, I'm starting fresh here, so I don't know whether or not the DID vault is created yet. If the DID vault already exists, then `sdsComm.CreateDIDVault() will not throw an error.

Let me know if there's some better way of doing this.